css/my_portfolio.css

```diff
@@ -99,3 +99,224 @@ header {
   top: 23em;
   left: 60%;
 }
+
+#work-section {
```

Comment:
- [ ] From the project specifications your work section should make use of grid layout.
`#work-section {
display: grid;
}`